run-utils.mjs

```diff
@@ -46,16 +46,19 @@ async function compileBootstrap() {
     return compileWithHostScheme('./schism/compiler.ss');
 }
 
-function make_compiler(compiler_bytes) {
-  return async function(bytes) {
-    const engine = new Schism.Engine;
-    const schism = await engine.loadWasmModule(await compiler_bytes());
-    engine.setCurrentInputPort(bytes);
-    let module_package = schism.exports['compile-stdin->module-package']();
-    module_package = engine.collect(module_package);
-    schism.exports['compile-module-package->stdout'](module_package);
-    return new Uint8Array(engine.output_data);
-  }
+function make_compiler(compiler_bytes, name) {
+    return async function(bytes, programName = "[unknown]") {
+        //console.info(`Compiling ${programName} with ${name}`);
+        const engine = new Schism.Engine;
+        const schism = await engine.loadWasmModule(await compiler_bytes());
+        engine.setCurrentInputPort(bytes);
+        // let module_package = schism.exports['compile-stdin->module-package']();
+        // module_package = engine.collect(module_package);
```

Comment:
Is this intentional?  With this change I am not sure what will call `collect()`.

Reply:
It was, but I probably should separate it out into a separate change (and it may not actually be a good idea), and maybe combine it with the change to lower the default heap size (see #56).

This GC was a quick hack because the compiler was using more memory than Wasm engines could support at the time. I realized the reason it was using so much memory was because I screwed up the arithmetic in `%alloc` and was allocating 8 times as much as I meant to. So, the hacky GC isn't really needed anymore.

I'd like to replace this with a proper collector (or just use Wasm GC once that's a thing). In the meantime though, we might as well keep this one so the existing collector code doesn't bitrot.